commands/errors.go

```diff
@@ -24,3 +27,13 @@ func checkTLSInsecure(gateway string, tlsInsecure bool) string {
 	}
 	return ""
 }
+
+//actionableErrorMessage print actionable error message based on APIError check
+func actionableErrorMessage(err error) error {
```

Comment:
A generic error message functions which can be used in all commands to print error message for generic APIError like 403, 500 etc